UnitTests/GitUI.Tests/UserControls/RevisionGrid/Graph/LaneInfoProviderTests.cs

```diff
@@ -16,7 +16,7 @@ public class LaneInfoProviderTests
         /// <summary>
         /// triples of merge subject, merged branch and destination branch ("into")
         /// for testing the LaneInfoProvider.References.MergeRegex
-        /// "(?i)^merged? (pull request (.*) from )?(.*branch |tag )?'?([^ ']*[^ '.])'?( of [^ ]*[^ .])?( into (.*[^.]))?\\.?$"
+        /// "(?i)^merged? (pull request (.*) from )?(.*branch |tag )?'?([^ ']*[^ '.])'?( of [^ ]*[^ .])?( into (.*[^.]))?\\.?$".
```

Comment:
The trailing dot is slightly confusing. Does the analyzer accept the dot on a new line?

Reply:
I cannot reproduce this easily, not sure what sequnce that is needed
It should report on the lines around this too
Can revert

Reply:
reverted